docs/managing-findings.md

```diff
@@ -0,0 +1,108 @@
+# Managing findings
+
+[TOC]
+
+# Findings
+
+A finding is the core unit of Semgrep's analysis. Findings are generated when a Semgrep rule matches a piece of code. After matching, a finding can make its way through 3 parts of the Semgrep ecosystem: [`semgrep`](https://github.com/returntocorp/semgrep), [`semgrep-action`](https://github.com/returntocorp/semgrep-action), and [semgrep.dev](https://semgrep.dev/).
+
+## Semgrep CLI
+
+Semgrep CLI findings are a specific rule matching a piece of code. Multiple rules can match the same piece of code, even if they are effectively the same rule. For example, consider the following rule and code snippet:
+
+```
+rules:
+- id: finding-test
+  pattern: $X == $X
+  message: Finding test 1
+  languages: [python]
+  severity: WARNING
+- id: finding-test
+  pattern: $X == $X
+  message: Finding test 2
+  languages: [python]
+  severity: WARNING
+```
+
+```
```

Comment:
Maybe label this snippet as `test.py`?

Reply:
I see your point, but I generally find it safe to assume that "consider the following rule and code snippet" corresponds to the snippets below and the `test.yaml` and `test.py` command. Am I being too generous?

Reply:
Probably just a me thing, I defer to you.